doc/data/messages/d/duplicate-argument-name/details.rst

```diff
@@ -0,0 +1,7 @@
+.. code:: python
+
+    >>> def get_fruits(apple, banana, apple):  # [duplicate-argument-name]
+    ...     pass
+    ...
+    File "<stdin>", line 1
```

Comment:
<img width="912" alt="Schermafbeelding 2022-04-01 om 09 06 33" src="https://user-images.githubusercontent.com/13665637/161212889-605270d5-8743-4382-9cec-b4c25bb289e1.png">

This is kind of duplicating what we already state above. We note that these raise SyntaxErrors and give this code example. Would you be okay with excluding this?

The `pylint-errors` project combines some of the data we originally provided so things that we have in our `Description` section can add up in their `Details` section.

Reply:
Okay, thanks for the advice. I will also remove it 😄